lib/foodcritic/command_line.rb

```diff
@@ -64,14 +67,35 @@ def initialize(args)
           @options[:search_grammar] = s
         end
         opts.on('-V', '--version',
-                'Display the foodcritic version.') do |v|
+                'Display the foodcritic version.') do |_v|
           @options[:version] = true
         end
         opts.on('-X', '--exclude PATH',
                 'Exclude path(s) from being linted.') do |e|
-          options[:exclude_paths] << e
+          @options[:exclude_paths] << e
+        end
+        opts.on('--config PATH',
+                'Path to foodcritic.yml file to use') do |f|
+          @options[:config] = f
         end
       end
+
+      cbs = @parser.permute(args)
+
+      get_config(cbs)
+
+      @options.merge!(fail_tags: @fc_config['fail_tags'].flatten.uniq) if @fc_config['fail_tags']
```

Comment:
I started with that, but keep getting errors if the values didn't exist in fc_config.  Plus, some are arrays which needed to be flattened, uniq'd, but not all.